src/index.tsx

```diff
@@ -310,10 +312,14 @@ function UseWalletProvider({
 
       // Initialize the web3-react connector if it exists.
       const web3ReactConnector = connector?.web3ReactConnector?.({
-        chainId,
+        supportedChains,
```

Comment:
Looking at the rest of connectors other than `frame`, `provided` and `injected`, they accept a `chainId` prop, in that case we might want to update those connectors to accept the `supportedChains` prop and pass as the chainId the first one as default? 

Was thinking how we can make those connectors support multichain but unless we provide a way for the user to provide which chain they would like to connect to, don't see an alternative.